Robot Code/src/org/usfirst/frc/team3316/robot/commands/turret/ZeroAngle.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc.team3316.robot.commands.turret;
+
+import org.usfirst.frc.team3316.robot.Robot;
+import org.usfirst.frc.team3316.robot.commands.DBugCommand;
+
+public class ZeroAngle extends DBugCommand
```

Comment:
This command is not necessary anymore, as the default mode for turret control is with the joystick.